src/lib/loaders/CopcStreamingLoader.integration.ts

```
@@ -0,0 +1,63 @@
+/**
+ * Integration code to add range request caching to CopcStreamingLoader
+ * This shows how to integrate the cache into the loader
+ */
+
+import type { RangeRequestCache } from './RangeRequestCache';
+import { RangeRequestCache } from './RangeRequestCache';
```

ESLint

[error] 'RangeRequestCache' is already defined.

(no-redeclare)